### DISCUSSION TITLE
Performance optimisation request [PERFORMANCE]

### DISCUSSION THREAD

**Discussion by TingTingShao** [aertslab/pycisTopic#119] [Q&A]
Dear, 

I only have snATAC-seq for now, and would like to try pycistopic to run. 

Original files: Fragments.tsv.gz
number of files/sample: 295
number of cells per file/sample: around 2000-3000 after QC
Number of features before excluding the blacklist: 6062095 regions (divided chromsome in 500bp bin size)

The files have been preprocess in snapATAC2 and want to try more features provided in cistopic and SCENICplus. 

I think the tutorial `Using pycisTopic on simulated melanoma data` is more suitful so far, but the count matrix is too large. Do you have any suggestions on how to process large count matrix or given without snRNA-seq data, which tutorial would be more suitable for me to get started? 

Looking forward to your reply!
thanks!
TingTing

**Comment by SeppeDeWinter** [comment]
Hi @TingTingShao 

I converted your issue to a discussion, because you are asking for advice (rather than describing a particular issue).
Sounds like an exciting project you have going on there.

At which particular step in the tutorial are you getting stuck?

All the best,

Seppe
